app/assets/javascripts/main.js

```diff
@@ -42,6 +43,20 @@ app.on("start", () => {
   app.vent = Backbone.Radio.channel("global");
 });
 
+app.on("start", () => {
+  // send WebGL analytics once per session
+  if (!window.sessionStorage.getItem("hasSentWebGLAnalytics")) {
+    try {
+      const webGLStats = getWebGLReport();
+      Request.sendJSONReceiveJSON("/api/analytics/webgl", {
+        data: webGLStats,
+      });
+      window.sessionStorage.setItem("hasSentWebGLAnalytics", true);
+    } catch (e) {
+      // pass
```

Comment:
No airbrake report? I think it would be good to know when the webgl report crashed? Otherwise the gathered data might be skewed.